tangos/input_handlers/ramsesHOP.py

```diff
@@ -0,0 +1,224 @@
+from more_itertools import always_iterable
+from ..util import proxy_object
+from itertools import chain
+from .pynbody import PynbodyInputHandler
+import numpy as np
+from ..log import logger
+from .. import config
+
+class RamsesHOPInputHandler(PynbodyInputHandler):
+    """ Handling Ramses outputs with HOP halo finding (Eisenstein and Hut 1998)"""
+    patterns = ["output_0????"]
+    auxiliary_file_patterns = ["grp*.tag"]
+
+    def match_objects(self, ts1, ts2, halo_min, halo_max, dm_only=False, threshold=0.005,
```

Comment:
Does this method copy/paste from somewhere else? It looks familiar. Is it possible to avoid the copy/paste and instead refactor to have this basic logic in one place?

Reply:
It resembles very much the same method defined in `input_handlers/pynbody.py` https://github.com/cphyc/tangos/blob/72e77fe8af987acf95e4d764ad0058e525bcbb0c/tangos/input_handlers/pynbody.py#L189-L212.
Note however that it has merely been moved from `tangos/input_handlers/pynbody.py` to tangos/input_handlers/ramsesHOP.py`.